src/utils/executor.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+
+const terminal: vscode.Terminal = vscode.window.createTerminal('Azure Functions');
```

Comment:
If the user closes/deletes the 'Azure Functions' terminal, then create new project will fail every time until they restart VS Code